src/Symfony/Component/AssetMapper/Tests/Command/AssetMapperCompileCommandTest.php

```diff
@@ -48,10 +48,9 @@ public function testAssetsAreCompiled()
             $this->filesystem->remove($targetBuildDir);
         }
         // put old "built" versions to make sure the system skips using these
-        $this->filesystem->mkdir($targetBuildDir);
-        file_put_contents($targetBuildDir.'/manifest.json', '{}');
-        file_put_contents($targetBuildDir.'/importmap.json', '{}');
-        file_put_contents($targetBuildDir.'/entrypoint.file6.json', '[]');
+        $this->filesystem->dumpFile($targetBuildDir.'/manifest.json', '{}');
```

Comment:
There was a call for mkdir - is it necessary to remain?

```suggestion
        $this->filesystem->mkdir($targetBuildDir);
        $this->filesystem->dumpFile($targetBuildDir.'/manifest.json', '{}');
```

Reply:
`dumpFile()` should already make sure, the directory we're dumping our file into exists.